test/index.js

```diff
@@ -4,12 +4,19 @@ var Wallet = require('../')
 var Thirdparty = require('../thirdparty.js')
 var ethUtil = require('ethereumjs-util')
 
-var fixturekey = Buffer.from('efca4cdd31923b50f4214af5d2ae10e7ac45a5019e9431cc195482d707485378', 'hex')
-var fixturewallet = Wallet.fromPrivateKey(fixturekey)
+var fixturePrivKey = 'efca4cdd31923b50f4214af5d2ae10e7ac45a5019e9431cc195482d707485378'
+var fixturePrivKeyStr = '0x' + fixturePrivKey
+var fixturePrivKeyBuffer = Buffer.from(fixturePrivKey, 'hex')
+
+var fixturePubKey = '5d4392f450262b276652c1fc037606abac500f3160830ce9df53aa70d95ce7cfb8b06010b2f3691c78c65c21eb4cf3dfdbfc0745d89b664ee10435bb3a0f906c'
+var fixturePubKeyStr = '0x' + fixturePubKey
```

Comment:
If `Buffer` isn't shortened, I'd have liked `PublicKey`, `PublicKeyString` and `PublicKeyBuffer` and same for private.